src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -180,7 +180,7 @@
 
 			var $today = $wrapper.find( '.tribe-events-present' ),
 				$mobile_trigger = $wrapper.find( '.mobile-trigger' ),
-				$tribe_grid = $wrapper.find( '#tribe-events-content > .tribe-events-calendar' );
+				$tribe_grid = $wrapper.find( document.querySelector( '#tribe-events-content .tribe-events-calendar' ) );
```

Comment:
Wrapping the result of `document.querySelector()` in a `find()` definitely jumps out at me as odd: what's the reason for doing this over say a straight `find()` (or wrapping the result of the querySelector call into a jQuery object)?